### DISCUSSION TITLE
TVB inputs generated by Cam-CAN branch

### DISCUSSION THREAD

**Discussion by dsodums** [McIntosh-Lab/tvb-ukbb#176] [Help]
Hi there,
I saw on other posts that there were substantial changes to the main branch which impacted the generation of the TVB inputs. Would this change also have impacted the Cam-CAN branch? I've recently processed an entire dataset using that branch so hoping not...
Thanks!
Devin

**Comment by justinliuyuwang** [comment]
No need to worry, it was only the main branch!

**Comment by dsodums** [comment]
Fantastic to hear, thanks for confirming!
